AxonDeepSeg/ads_utils.py

```diff
@@ -0,0 +1,72 @@
+import os
+import sys
+import raven
+
+
+def init_ads():
+    """ Initialize ads for typical terminal usage
+    :return:
+    """
+    bugTracking = True
+
+    init_error_client(bugTracking)
+
+
+def init_error_client(bugTracking):
+    """ Send traceback to neuropoly servers
+    :return:
+    """
+
+    if bugTracking:
+
+        try:
+
+            client = raven.Client(
+                        "https://e04a130541c64bc9a64939672f19ad52@sentry.io/1238683",
+                        processors=(
+                            'raven.processors.RemoveStackLocalsProcessor',
+                            'raven.processors.SanitizePasswordsProcessor')
+                            )
+
+            traceback_to_server(client)
+
+            old_exitfunc = sys.exitfunc
+
+            def exitfunc():
+                sent_something = False
+                try:
+                    # implementation-specific
+                    import atexit
+                    for handler, args, kw in atexit._exithandlers:
+
+                        if handler.__module__.startswith("raven."):
+                            sent_something = True
+
+                except:
+                    pass
+                old_exitfunc()
+
+                print ("Note: you can opt out of Sentry reporting by "
```

Comment:
@jcohenadad I actually just noticed that this message prints not only for when there was a bug, but when ADS exits successfully as well. I'll have to explore this a bit more. Maybe I should just wait for our Python 3.5x or 6.x upgrade, as the SCT+raven implementation for this feature might work then.

Reply:
oops! maybe you can try to reach out to sentry's forum or post on their github? it would be nice to implement a solution directly on this PR.